services/jobpost/jobpost.go

```diff
@@ -0,0 +1,58 @@
+package service
+
+import (
+	"fmt"
+
+	"github.com/hngprojects/hng_boilerplate_golang_web/internal/models"
+	"github.com/hngprojects/hng_boilerplate_golang_web/utility"
+	"gorm.io/gorm"
+)
+
+func ValidateCreateJobPost(req models.JobPost) error {
```

Comment:
if there are unique fields, validate if the job exists in db here, as this is a redundant function if you do the first review, check other services for reference

Reply:
for this job there are no current unique fields.